### ISSUE TITLE
Why the html/body font-size is 14px instead of 16px?

### ISSUE THREAD

**Issue by Puspendert** [fomantic/Fomantic-UI#1214]
In most of the modern websites(at least top most) the font-size is ~16px but why `fomantic` is still on `14px`? Even most of the browsers use 16px as their default.
Any specific reason for sticking onto 14px?

**Comment by donaggio** [comment]
You can change it as you please, it's just a LESS variable to override in your theme config.

**Comment by exoego** [comment]
It is true that 16px or more is common nowadays.
But increasing font size in v2.x is a breaking change.

**Comment by Puspendert** [comment]
@donaggio how? Should I follow [this](https://medium.com/@fomantic/setting-up-fomantic-ui-with-react-f701b68f736c) blog or is there any other easy way?

**Comment by Puspendert** [comment]
@exoego is this planned in any milestone? v3 or upper?

**Comment by donaggio** [comment]
@Puspendert to adjust the base font size just override @fontSize in your src/site/globals/site.variables as explained [here](https://fomantic-ui.com/usage/theming.html)

**Comment by lubber-de** [comment]
@Puspendert 
We can discuss to change the default in a new minor version like 2.9
@fomantic/maintainers What do you think?

To satisfy your need on your own now do the following:

- Change the two values from `14px` to `16px` in

https://github.com/fomantic/Fomantic-UI/blob/662c43f82de2b11677c1a7dd7fa5052181b4cdc5/src/themes/default/globals/site.variables#L33-L38

- Run `npx gulp build` in your fomantic root dir. You may have to run `npm install` beforehand
- Use the newly created semantic.css in the dist-folder whereever you need it

**Comment by Puspendert** [comment]
Will this setup work for a react application built using `create-react-app`.
If maintainers could verify I can wait for 2.9 version

**Comment by hammy2899** [comment]
@Puspendert Yes you can change the `@fontSize` variable to change it now, this guide should help you setup your react project with theming support https://medium.com/@fomantic/setting-up-fomantic-ui-with-react-f701b68f736c

@lubber-de @fomantic/maintainers we could do this for 2.9 however that will most likely be around March 2020.

**Comment by exoego** [comment]
I understand that introducing breaking change in minor versions are reasonable for some cases, e.g. security issue, bug, or 
But why we introduce breaking change in font in v2.9 despite users can configure font size ?
I think most users expects minor versions as drop-in, and will get surprised if font size increased and their design get broken.

**Comment by hammy2899** [comment]
@exoego Agreed however with the current version scheme SUI/FUI use 2.9 would be a major release which would allow this kind of change. That being said this definitely needs more discussion because as you said a user can change it fairly easily.

**Comment by Puspendert** [comment]
@hammy2899 @lubber-de  I tried following the https://medium.com/@fomantic/setting-up-fomantic-ui-with-react-f701b68f736c . `npm install --save fomantic-ui semantic-ui-react` does work but since I have created my project using `yarn`, the command `yarn add fomantic-ui semantic-ui-react` doesn't run as expected. `npm install` asks for `? Where should we put Semantic UI inside your project?` but `yarn` does it automatically and that too into the `node_module` folder.

Is this the issue with `yarn` or the `fomantic-ui`?
